packages/manager/apps/web-office/src/hooks/__test__/useOfficeParentTenant.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { vi, describe, expect } from 'vitest';
+import '@testing-library/jest-dom';
+import { renderHook, waitFor } from '@testing-library/react';
+import { parentTenantMock } from '@/api/_mock_';
+import { useOfficeParentTenant } from '@/hooks';
+import { wrapper } from '@/utils/test.provider';
+
+const useParamsMock = vi.hoisted(() => ({
+  useParams: vi.fn(),
+}));
+
+vi.mock('react-router-dom', async (importActual) => {
+  const actual = await importActual<typeof import('react-router-dom')>();
+  return {
+    ...actual,
+    useParams: useParamsMock.useParams,
```

Comment:
it's better to do `vi.mocked(useParams).mockReturnedValue(...)` than having to redeclare a whole `vi.mock('react-router-dom', ...)`

Reply:
It's fixed.